coder_sniffer/Drupal/Test/Semantics/RemoteAddressUnitTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Unit test for Drupal.Semantics.RemoteAddress sniff.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+/**
+ * Test class for Drupal.Semantics.RemoteAddress sniff.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+class Drupal_Sniffs_Semantics_RemoteAddressUnitTest extends CoderSniffUnitTest
```

Comment:
needs to be updated for Coder 8.x-3.x. Can you update this PR and switch the base branch to 8.x-3.x?

Reply:
The class name should be changed to be just RemoteAddressUnitTest.

Reply:
Done!